crates/bevy_diagnostic/src/system_information_diagnostics_plugin.rs

```diff
@@ -122,6 +126,7 @@ pub mod internal {
         };
 
         info!("{:?}", info);
+        commands.insert_resource(info);
```

Comment:
Can we somehow do this at app build time? This way, we can use this struct in Startup without having to worry about system ordering at all.